### PR TITLE
graphs.py new file + some changes in homepage.py layout !

### PR DESCRIPTION
**Pull request by nrslt**
Hi guys,
I just added a graphs.py file that will hold all plotly figure objects generated for various indicators (depending on the DS team: meteorological data, risk evaluation, historical information ...)
I also modified the general layout of the homepage: since we expect to find a large enough map when landing on homepage + possibility to slide/change map view + graphs, that represented a lot of information so I decided to put the graphs at the bottom of the page for ease of reading

let me know what you think, of course the layout can be modified and I let @pechouc add the slider

### CODE REVIEW

**Comment by nrslt** [comment]
I just did the small modifications for the flake8 checks to go smoothly!

File: app/homepage.py [hunk 1]
```diff
@@ -34,51 +34,49 @@
...
-        dbc.Col(dcc.Dropdown(id='user_department_input',
-                            options=[
-                            {'label': 'Ardèche', 'value': 'Ardèche'},
-                            {'label': 'Gard', 'value': 'Gard'},
-                            {'label': 'Landes', 'value': 'Landes'}
-                            ]))
-        ]),
-    dbc.Row(
+         dbc.Col(dcc.Dropdown(id='user_department_input',
+                              options=[{'label': 'Ardèche', 'value': 'Ardèche'},
```
Comment (VincGargasson) [inline app/homepage.py:40]: @nrslt  You will include later the logic that zoom on the department selected by the user?

**Review by VincGargasson — CHANGES_REQUESTED** [review]
LGTM ! @nrslt could you add a pic to your PR to see what it looks like while running? thanks !

File: app/homepage.py (hunk 1 above)
Comment (nrslt) [reply app/homepage.py:40]: @VincGargasson definitely! As I explained yesterday, the container/row/column logic that is inherent of [dash bootstrap components](https://dash-bootstrap-components.opensource.faculty.ai/) is really practical and easy to remodel.
Here is what it looks like at the moment:
Capture d’écran, le 2020-10-15 à 08 30 45
Capture d’écran, le 2020-10-15 à 08 31 13

Comment (nrslt) [reply app/homepage.py:40]: @VincGargasson As for the department zoom, for now I just added the same map as alerts.py. We will need to add a callback element that links the user choice of department with a new map, centered on that department.

Again, the idea here is more about quickly setting a template for us to keep on working on the homepage (for @pechouc to work on the slider, which has its own available slot, and for you to work on the map)
Let me know if this logic makes sense and if you'd like any modification before approving this PR!

Capture d’écran, le 2020-10-15 à 08 38 46


cheers

**Review by nrslt — COMMENTED** [review]
so if it's ok with you, I propose we set this basic layout in order to all have a common ground to work on!

**Review by pechouc — APPROVED** [review]
I might join the conversation a bit late but I agree with the idea of setting a preliminary layout and build upon it in parallel streams 😊

**Review by nrslt — COMMENTED** [review]
@VincGargasson is it ok with you?

**Review by Akilditu — APPROVED** [review]
@nrslt thanks for the PR ! It's indeed an interesting baseline to experience pages structure.

**Review by VincGargasson — APPROVED** [review]

**Pull request merged**